archive/j/java/BinarySearch.java

```diff
@@ -0,0 +1,33 @@
+/* Program: Binary Search
+ * Input: Number of elements, element's values, value to be searched
+ * Output:Position of the number input by user among other numbers*/
+class BinarySearch
+{
+   public static void main(String args[])
+   {
+      int counter, item, first, last, middle;
+      int array[] = {1, 4, 5, 11, 12};
```

Comment:
This should come out of `args` as a string which you can convert into a list of numbers.